app/_data/docs_nav_studio_1.2.x.yml

```
@@ -0,0 +1,34 @@
+- title: Kong Studio
+  items:
+    - text: Release Notes
+      url: /release-notes
+    - text: Changelog
+      url: /studio/changelog
+      absolute_url: true
+- title: Getting Started
+  url: /overview
+  items:
+    - text: Download & Install Kong Studio
+      url: /download-install
+    - text: User Interface
+      url: /user-interface
+    - text: Editing Spec Files
+      url: /editing-specs
+    - text: Enabling Live Preview Mode
+      url: /live-preview
+    - text: Debugging with Insomnia
+      url: /debugging-with-insomnia
+    - text: Git Sync with Studio
+      url: /git-sync
+    - text: Deploy to Dev Portal
+      url: /deploy-to-dev-portal
+    - text: Linting in Kong Studio
+      url: /linting
+- title: Guides & References
+  items:
+    - text: API Design Best Practices
+      url: /best-practices
+    - text: Generating Kong Declarative Config with Kong Studio
+      url: /dec-conf-studio
+    - text: OpenAPI GraphQL Support
+      url: /graphql
```

@nijikokun I think this is causing a newline error

https://travis-ci.org/Kong/docs.konghq.com/builds/638583743#L369

```suggestion
      url: /graphql

```